tests/post/steps/test_logs.py

```diff
@@ -2,6 +2,9 @@
 
 from pytest_bdd import scenario, then
 
+from tests import kube_utils
+from tests import utils
```

Comment:
```suggestion
from tests import kube_utils, utils
```
Please use only one line here